generate.go

```diff
@@ -25,6 +25,11 @@ func main() {
 				sk_anyvendor.EnvoyValidateProtoMatcher.Package: sk_anyvendor.EnvoyValidateProtoMatcher.Patterns,
 			},
 		},
+		ValidationSchemaOptions: &schemagen.ValidationSchemaOptions{
+			// Path to where test CRDs are stored
+			CrdDirectory:   "test/mocks/crds",
+			JsonSchemaTool: "protoc",
```

Comment:
It's probably unnecessary to expose this to the user. I had done this to make testing easier while I was switching back and forth between the implementations. Open to thoughts

Reply:
I think it's fine to leave exposed